cmd/minishift/cmd/hostfolder/list.go

```diff
@@ -17,38 +17,68 @@ limitations under the License.
 package hostfolder
 
 import (
+	"fmt"
 	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/libmachine/drivers"
 	"github.com/minishift/minishift/cmd/minishift/cmd/util"
 	"github.com/minishift/minishift/cmd/minishift/state"
 	"github.com/minishift/minishift/pkg/minikube/constants"
-	hostfolderActions "github.com/minishift/minishift/pkg/minishift/hostfolder"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/cobra"
+	"os"
+	"text/tabwriter"
 )
 
-var hostfolderListCmd = &cobra.Command{
+var listCmd = &cobra.Command{
 	Use:   "list",
 	Short: "Lists the defined host folders.",
-	Long:  `Lists an overview of the defined host folders that can be mounted to a running OpenShift cluster.`,
+	Long:  `Lists an overview of the defined host folders that can be mounted into the Minishift VM.`,
```

Comment:
Why minishift vm here, while in previous files this was changed to openshift cluster...

Reply:
Other way around. I tried to changed it from OpenShift cluster to Minishift VM and missed to do so in the mount command. I believe that the right way to describe this is to mount something into the VM or into the filesystem of the VM. OpenShift is not even involved here and talking about OpenShift cluster might also create wrongly some association to persistent volumes.

Reply:
I think Minishift VM is more transparent.